### ISSUE TITLE
Unexpected Token Error on prebid latest version while old version working fine.

### ISSUE THREAD

**Issue by Vikramvijay6** [prebid/Prebid.js#3803]

**Comment by mkendall07** [comment]
@Vikramvijay6 
please provide more details by filling out the template.

**Comment by Vikramvijay6** [comment]
Hi Matt

Below is the URL you can check the same :

https://mobileads.indiatimes.com/ET/Mobile_Ads/Prebid/prebid2.13.0.js

Regards
Vikram Vijay

**Comment by jsnellbaker** [comment]
@Vikramvijay6 

Can you list out which adapters and modules you had used when building this file?  I tried to make a basic build with just a few adapters and modules, and it worked fine with a test page I loaded.

That said - I am seeing the error you noted when attempting to use your file, so I wanted to replicate making a build with the same parameters you used to investigate further.

**Comment by Vikramvijay6** [comment]
@ Team

The adaptor which I have selected is below in the file and no module I have
selected:

appnexus
andbeyond
criteo
pubmatic
rubicon project

Regards
Vikram Vijay

**Comment by jsnellbaker** [comment]
@Vikramvijay6 Thanks for providing the modules.  I generated build files using this list through the download link on prebid.org/download.html as well as through a local version of the project.  In both cases, the generated files worked fine with the hello_world test page.

Have you seen this behavior occur with more recent versions of Prebid.js?  If so, which method from the two listed above are you using to generate the file?  Additionally, how are you going about uploading the file onto your server, is it going directly to the server via some sftp (or something equivalent) or is it through a third party service?  

When I looked through the 'bad' file, I saw several areas throughout the file that were syntactically incorrect that took the form of transforming some sequence of characters into another set; so I think it's something with the upload process, but I'm not sure what would do this type of thing.
